build.gradle

```diff
@@ -13,7 +13,7 @@ ext {
     wordPressUtilsVersion = '2.7.0'
     wordPressLoginVersion = '0.19.0'
     aztecVersion = 'v1.6.0'
-    gutenbergMobileVersion = 'v1.83.0'
+    gutenbergMobileVersion = '5210-20d7e4c6eba8d46853379ce1bf4b061c61f22b83'
```

Comment:
Friendly reminder we'll need to update this to the release tag before merging.

Reply:
Reference updated to release tag in https://github.com/wordpress-mobile/WordPress-Android/pull/17327/commits/b2db07b32d6b5db1c0c06e29da7db1fdc9d1faae.